staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -161,15 +162,22 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 		}
 
 		trace.Step("About to store object in database")
-		result, err := finishRequest(timeout, func() (runtime.Object, error) {
+		requestFunc := func() (runtime.Object, error) {
 			return r.Create(
 				ctx,
 				name,
 				obj,
 				rest.AdmissionToValidateObjectFunc(admit, admissionAttributes, scope),
 				options,
 			)
-		})
+		}
+		result, err := finishRequest(timeout, requestFunc)
```

Comment:
Do we need to do this in the update handler, too? I think we might?

Reply:
I have this in the update handler and the patch handler as well